its/ruling/src/integrationTest/resources/sources/docker/parsingError/SONARIAC-1485.dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM debian:stretch-slim
+
+ENV BDBVER db-4.8.30.NC
+RUN wget --quiet https://download.oracle.com/berkeley-db/$BDBVER.tar.gz; \
+        echo "Done"
```

Comment:
Can this be also added to the Unit Test for the check? There we can verify the range the issue raised on in more detail.